src/Illuminate/Redis/Connectors/PhpRedisConnector.php

```diff
@@ -151,6 +155,10 @@ protected function createRedisClusterInstance(array $servers, array $options)
             $parameters[] = $options['password'] ?? null;
         }
 
+        if (version_compare(phpversion('redis'), '5.3.2', '>=')) {
```

Comment:
Did you mean `5.3.0` here?

Reply:
No, it will be part of the 5.3.2 release, it's already in develop and working, but not released, yet.